<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inventario</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../iconos.qrc">
    <normaloff>:/assets/icons/inventario.png</normaloff>:/assets/icons/inventario.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="progressB_barra">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>691</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_volver">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>380</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 115, 255);
color: rgb(255, 255, 255);
font: 75 10pt &quot;Arial Black&quot;;
border-radius: 8px;
padding: 6px 12px;</string>
    </property>
    <property name="text">
     <string>Volver al Menú</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tb_productos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>681</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Arial&quot;;</string>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="lb_fondoblanco">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>681</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.6875, y1:0.216, x2:1, y2:0, stop:1 rgba(255, 255, 255, 255));
border-radius: 9px;
padding: 10px 12px;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_blancocomentario">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>380</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.6875, y1:0.216, x2:1, y2:0, stop:1 rgba(255, 255, 255, 255));
border-radius: 9px;
padding: 10px 12px;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_inventario">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>681</width>
      <height>58</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 115, 255);
color: rgb(255, 255, 255);
font: 87 12pt &quot;Arial Black&quot;;
border-radius: 10px; 
padding: 8px 12px;
</string>
    </property>
    <property name="text">
     <string>Inventario</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkB_todos">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Todos los Productos</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkB_sin">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Productos Sin Stock</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkB_con">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Productos Con Stock</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_exportar">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>90</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 115, 255);
color: rgb(255, 255, 255);
font: 75 10pt &quot;Arial Black&quot;;
border-radius: 8px;
padding: 6px 12px;</string>
    </property>
    <property name="text">
     <string>Exportar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_fecha">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Consolas&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHerramientas">
    <property name="title">
     <string>Herramientas</string>
    </property>
    <addaction name="actionSistema"/>
    <addaction name="actionClaro"/>
    <addaction name="actionOscuro"/>
   </widget>
   <widget class="QMenu" name="menuM_dulos">
    <property name="title">
     <string>Módulos</string>
    </property>
    <addaction name="actionReporte_Ventas"/>
    <addaction name="actionInventario"/>
    <addaction name="actionActualizaci_n_de_Productos"/>
    <addaction name="actionLista_de_Distribuidores"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <widget class="QMenu" name="menuCredenciales_API">
    <property name="title">
     <string>Credenciales API</string>
    </property>
    <addaction name="actionWooCommerce"/>
    <addaction name="actionCredenciales_API"/>
   </widget>
   <addaction name="menuCredenciales_API"/>
   <addaction name="menuM_dulos"/>
   <addaction name="menuHerramientas"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSistema">
   <property name="text">
    <string>Sistema</string>
   </property>
  </action>
  <action name="actionClaro">
   <property name="text">
    <string>Claro</string>
   </property>
  </action>
  <action name="actionOscuro">
   <property name="text">
    <string>Oscuro</string>
   </property>
  </action>
  <action name="actionReporte_Ventas">
   <property name="text">
    <string>Reporte Ventas</string>
   </property>
  </action>
  <action name="actionInventario">
   <property name="text">
    <string>Inventario</string>
   </property>
  </action>
  <action name="actionActualizaci_n_de_Productos">
   <property name="text">
    <string>Actualización de Productos</string>
   </property>
  </action>
  <action name="actionLista_de_Distribuidores">
   <property name="text">
    <string>Lista de Distribuidores</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
  <action name="actionWooCommerce">
   <property name="text">
    <string>WooCommerce</string>
   </property>
  </action>
  <action name="actionCredenciales_API">
   <property name="text">
    <string>Credenciales API</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../iconos.qrc"/>
 </resources>
 <connections/>
</ui>
